sdk/src/client/api/block_builder/input_selection/transition.rs

```diff
@@ -55,11 +55,19 @@ impl InputSelection {
         // Remove potential sender feature because it will not be needed anymore as it only needs to be verified once.
         let features = input.features().iter().filter(|feature| !feature.is_sender()).cloned();
 
-        let builder = AccountOutputBuilder::from(input)
+        let mut builder = AccountOutputBuilder::from(input)
             .with_account_id(account_id)
             .with_foundry_counter(u32::max(highest_foundry_serial_number, input.foundry_counter()))
             .with_features(features);
 
+        if input.is_block_issuer() {
```

Comment:
Should we just do it all the time?

Reply:
Can we use this Mana still for an allotment then?

Reply:
Allotment to the same account you mean? That's a good question, I'll ask

Reply:
Yes, I think we can but my question was more like if ISA still can do it if we set it here on the output already

Reply:
But isn't that something we probably want most of the time (to allot to this account)?

Reply:
Maybe at least put a TODO or a debug log, otherwise might be hard to find later why this amount is not available

Reply:
https://github.com/iotaledger/iota-sdk/issues/1918

Reply:
Added TODO in the code as well